js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/invertedIndex/useCreateInvertedIndex.ts

```diff
@@ -253,10 +253,18 @@ const fieldSchema: any = Yup.object().shape({
 });
 const schema = Yup.object({
```

Comment:
We are using [Formik](https://github.com/jaredpalmer/formik), which supports Yup out of the box. JSONeditor uses JSON schema.

I will check out the library mentioned and see if it will fit the use case, but forms and JSONs have different use-cases, so it might not always work out.


but in any case I agree, we need to consolidate the validation solutions; will keep a track of this